lib/splitclient-rb/sse/workers/splits_worker.rb

```diff
@@ -32,25 +32,44 @@ def stop
           SplitIoClient::Helpers::ThreadHelper.stop(:split_update_worker, @config)
         end
 
-        def add_to_queue(change_number)
-          @config.logger.debug("feature_flags_worker add to queue #{change_number}")
-          @queue.push(change_number)
+        def split_update(notification)
+          if @splits_repository.get_change_number() == notification.data['pcn']
+            begin
+              @new_split = JSON.parse(SplitIoClient::Helpers::DecryptionHelper.get_encoded_definition(notification.data['c'], notification.data['d']), symbolize_names: true)
```

Comment:
When you declare a variable with `@` prefix you are declaring an instant variable and is available to all methods within the class. 
In this case you should to declare without `@` given that you need a local variable.

Reply:
will remove it